prometheus/update.sh

```diff
@@ -15,7 +19,7 @@ while getopts ":i:" opt; do
   esac
 done
 
-[[ -z "${PROMETHEUS_SHA}" ]] && PROMETHEUS_SHA="$(grep '%global git_commit ' prometheus.spec | cut -d' ' -f3)"
+PROMETHEUS_SHA=${PROMETHEUS_SHA:-"$(grep '%global git_commit ' prometheus.spec | cut -d' ' -f3)"}
```

Comment:
and here

Reply:
that's not the focus of this pr. I only changed strictly what was necessary for update.sh to succeed with my changes. Further improvements can be done in another PR.